site/glossary.html

```diff
@@ -14,12 +17,12 @@
 
     <header id="top">
       <nav>
-        <ul>
+        <!-- <ul>
           <li><a href="index.html">Home</a></li>
           <li><a href="links.html">Links</a></li>
           <li><a href="glossary.html">Glossary</a></li>
           <li><a href="survey.html">Survey</a></li>
-        </ul>
+        </ul> -->
       </nav>
```

Comment:
This could be cleaned up.
```suggestion
<nav>
  <script src="scripts/navigation.js"></script>
</nav>
```